ludicrousdb/includes/class-ludicrousdb.php

```diff
@@ -631,8 +647,8 @@ public function db_connect( $query = '' ) {
 				return $this->bail( "No database servers were found to match the query. ({$this->table}, {$dataset})" );
 			}
 
-			if ( ! isset( $unique_servers ) ) {
-				$unique_servers = $tries_remaining;
+			if ( null === $this->unique_servers ) {
```

Comment:
How do you feel about `is_null()` instead of `null ===`?

Not lobbying for a change, necessarily. Aiming for consistency.

Reply:
Resolved.